src/v0/destinations/am/transform.js

```diff
@@ -297,6 +297,15 @@ const identifyBuilder = (message, destination, rawPayload) => {
       }
     });
   }
+  // update identify call request with unset fields
+  const fieldsToUnset = get(message, 'integrations.Amplitude.fieldsToUnset');
```

Comment:
can you document the format of data here

Reply:
done